test/test_helper.rb

```diff
@@ -18,3 +13,29 @@
   config.extend Yast::I18n  # available in context/describe
   config.include Yast::I18n # available in it/let/before/...
 end
+
+if ENV["COVERAGE"]
+  require "simplecov"
+  SimpleCov.start do
+    add_filter "/test/"
+  end
+
+  # For coverage we need to load all ruby files
+  # Note that clients/ are excluded because they run too eagerly by design
+  fs = Dir["#{srcdir}/**/*.rb"]
+  fs.delete_if { |f| f.start_with? "#{srcdir}/clients/" }
+  # HACK: this would BuildRequire yast2-slp so defer this
+  fs.delete_if { |f| f.end_with? "/SourceManagerSLP.rb" }
+  fs.each do |f|
+    require_relative f
+  end
+
+  # use coveralls for on-line code coverage reporting at Travis CI
+  if ENV["TRAVIS"]
+    require "coveralls"
+    SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
```

Comment:
There is a [new simplified API](https://github.com/colszowka/simplecov#using-multiple-formatters) for multiple formatters:
```ruby
SimpleCov.formatters = [
  SimpleCov::Formatter::HTMLFormatter,
  Coveralls::SimpleCov::Formatter
]
```

We should update this also in the other modules....